### PR TITLE
Remove exception if no key is found in the m3u8

### PR DESCRIPTION
**Pull request by gdoucet**

### CODE REVIEW

**Comment by gdoucet** [comment]
This commit fixes #71

File: toutv/dl.py [hunk 1]
```diff
@@ -297,10 +302,11 @@ def download(self):
         self._logger.debug('parsed M3U8 file: {} total segments'.format(self._total_segments))
 
         # get decryption key
-        uri = self._segments[0].key.uri
-        r = self._do_request(uri)
-        self._key = r.content
-        self._logger.debug('decryption key: {}'.format(self._key))
+        if self._segments[0].key:
```
Comment (simark) [inline toutv/dl.py]: I have so many questions:

* So there is no more decryption key?
* That means that the segments are now sent unencrypted?
* Do you know if it's like this for all the videos?

File: toutv/dl.py [hunk 2]
```diff
@@ -297,10 +302,11 @@ def download(self):
...
+            uri = self._segments[0].key.uri
+            r = self._do_request(uri)
+            self._key = r.content
+            self._logger.debug('decryption key: {}'.format(self._key))
```
Comment (simark) [inline toutv/dl.py]: Can you put an else block with a logging message?  Something like:

    self._logger.debug('no decryption key found')

**Comment by simark** [comment]
Oh and thank you very much for fixing this.

File: toutv/dl.py (hunk 1 above)
Comment (gdoucet) [inline toutv/dl.py]: Not sure if it is for all videos now. Check with Infoman and Les pêcheurs, both were unencrypted. Each segments seem to be in clear now.

File: toutv/dl.py (hunk 2 above)
Comment (gdoucet) [inline toutv/dl.py]: Added the commit for the logging.

File: toutv/dl.py (hunk 1 above)
Comment (simark) [inline toutv/dl.py]: Ok, I think it's a good idea to keep the possibility there being a key, in case it's just temporary.

**Pull request merged**